cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -199,6 +202,7 @@
     private KafkaAuthorization authorization;
     private Set<String> externalAddresses = new HashSet<>();
     private KafkaVersion kafkaVersion;
+    private CertAndKeySecretSource secretSource = null;
```

Comment:
try to be more specific than just `secretSource`, this Secret is about containing broker certificates.

Reply:
This was proposed by @scholzj